src/app/organization/organization.service.ts

```diff
@@ -291,6 +291,34 @@ export class OrganizationService {
     return this.http.get('/holidays', { params: httpParams });
   }
 
+  /**
+   * Get Holiday data.
+   * @param holidayId Holiday Id of holidays.
+   * @returns {Observable<any>} Holiday data.
+   */
+  getHoliday(holidayId: string): Observable<any> {
+    return this.http.get(`/holidays/${holidayId}`);
+  }
+
+  /**
+   * Activate Holiday.
+   * @param holidayId Holiday Id to be activated.
+   * @returns {Observable<any>} Resource Id.
+   */
+  activateHoliday(holidayId: string): Observable<any> {
+    const httpParams = new HttpParams().set('command', 'activate');
```

Comment:
Please carefully check API parameters
```suggestion
    const httpParams = new HttpParams().set('command', 'Activate');
```